### DISCUSSION TITLE
Implement ramp up bucket

### DISCUSSION THREAD

**Discussion by Rutenioh** [vladimir-bukhtoyarov/bucket4j#263] [Q&A] [answered]
Hello, 
awesome lib! I love it.
We are trying to implement a ramp up/down bucket. For example, configure a bucket with N tokens but it reaches gradually the N tokens after a time period.
I don't know if there is a configuration to simulate this behaviour.

**Comment by vladimir-bukhtoyarov** [comment, marked as answer]
Hello @Rutenioh 

There is something that looks similar, try to play with [specifying initial amount of tokens](https://bucket4j.com/7.5.0/toc.html#specifying-initial-amount-of-tokens).
